straxen/mini_analysis.py

```diff
@@ -66,7 +67,12 @@ def wrapped_f(context: strax.Context, run_id: str, **kwargs):
             if "to_pe" in parameters and "to_pe" not in kwargs:
                 to_pe = context.config["gain_model"]
                 if isinstance(to_pe, str):
-                    to_pe = straxen.URLConfig.evaluate_dry(to_pe, run_id=run_id)
+                    if "corrections_run_id" in kwargs:
```

Comment:
This part should be a stand-alone function called `process_to_pe` or other proper names. For straxen, this function can be as what it was. But for fuse, you can process `corrections_run_id` in it.